src/Fixer/FunctionNotation/VoidReturnFixer.php

```diff
@@ -203,8 +203,6 @@ private function hasVoidReturn(Tokens $tokens, $startIndex, $endIndex)
     }
 
     /**
-     * Apply the fix to the function definition.
```

Comment:
yeah, I would delete them all, but than I again I know I'm a bit of a minimalist on these things